lotto.js

```diff
@@ -0,0 +1,156 @@
+let pcArray = [];
+const createNum = document.querySelector(".num-create-btn");
+createNum.disabled = true;
+
+createNum.style.background = "#A0B3E0";
+
+function randomNum() {
+  let array = [];
+  while (array.length < 6) {
+    let num = parseInt(Math.random() * (45 - 1) + 1);
+    if (!array.includes(num)) {
+      array.push(num);
+    }
+  }
+  return array.sort((a, b) => a - b); //오름차순 정렬
+}
+
+function lotto() {
+  const winningNum = randomNum();
+  create(winningNum);
+  showResult(winningNum);
+}
+
+function create(winningNum) {
+  const wrapperResult = document.createElement("div");
+  wrapperResult.className = "wrapper-result";
+
+  const winResult = document.createElement("div");
+  winResult.className = "winning-result";
+  winResult.innerText = "당첨 결과 확인";
+
+  const weekResult = document.createElement("div");
+  weekResult.className = "winning-result-text1";
+  weekResult.innerText = "이번 주 당첨 번호:";
+
+  const wrapperBall = document.createElement("div");
+  wrapperBall.className = "wrapper-ball";
+
+  winningNum.forEach((num, index) => {
+    const ball = document.createElement("div");
+    ball.className = `ball ball-0${index + 1}`;
+
+    const ballNum = document.createElement("div");
+    ballNum.className = "ball-number";
+    ballNum.innerText = num;
+
+    ball.appendChild(ballNum); //밑에거랑 순서 바뀌면 안됨
+    wrapperBall.appendChild(ball);
+  });
+
+  wrapperResult.appendChild(winResult);
+  wrapperResult.appendChild(weekResult);
+  wrapperResult.appendChild(wrapperBall);
+
+  const plusResult = document.querySelector(".plus-result-01");
+  plusResult.innerHTML = ""; //초기화
+  plusResult.appendChild(wrapperResult);
+}
+
+function purchase() {
+  pcArray = [];
+
+  const ea = document.querySelector("#purchase-ea").value;
+
+  const price = document.querySelector(".total-price");
+  if (ea > 0) {
+    price.innerText = `총 금액: ${(
+      ea * 1000
+    ).toLocaleString()}원 (1장당 1,000원)`;
+  }
+
+  for (let i = 0; i < ea; i++) {
+    createNum.disabled = false;
+    if (!createNum.disabled) {
+      createNum.style.background = "#4B89DC";
+    }
+    let pcArray2 = [];
+
+    while (pcArray2.length < 6) {
+      let num = parseInt(Math.random() * (45 - 1) + 1);
+      if (!pcArray2.includes(num)) {
+        pcArray2.push(num);
+      }
+    }
+
+    pcArray2.sort((a, b) => a - b);
+    pcArray.push(pcArray2);
+  }
+  pcShow(pcArray);
+  if (pcArray.length === 0) {
+    alert("로또를 구매해주세요");
+  }
+}
+
+function pcShow(pcNum) {
```

Comment:
`pcShow` 라는 함수명과 `pcNum` 파라미터를 좀 더 의미있는 이름으로 지어주면 좋을 거 같아요.
```suggestion
function showLottoNumbers(lottoNumbers) {
```

컴퓨터가 생성한 로또 번호를 보여주는 함수로 보여서 위와 같이 이름을 지었습니다. 조금 더 의미있어 보이지 않나요?
그리고 `pcNum` 이라는 변수명은 **컴퓨터가 생성한 숫자** 정도로 이해가 되는데, 로또 번호는 반드시 6개가 생성되기 때문에 `numbers` 라고 지칭하면 좋을 거 같아요. 여기서 *로또 번호들* 이기에 `lottoNumbers` 라고 만들어봤어요.

처음에 이름짓는게 어렵지만, 다른 사람이 내 코드를 보고 이해할 수 있을 정도로 변수명을 지어주면 좋습니다!

Reply:
다음부터는 변수명을 좀 더 신경쓰도록 하겠습니다!